ironfish/src/rpc/routes/wallet/getNotes.ts

```diff
@@ -48,8 +50,13 @@ router.register<typeof GetAccountNotesStreamRequestSchema, GetAccountNotesStream
       const transaction = await account.getTransaction(transactionHash)
       Assert.isNotUndefined(transaction)
 
+      const asset = await node.chain.assets.get(note.assetIdentifier())  
+
+      Assert.isNotUndefined(asset)
```

Comment:
You can't guarantee the asset is found because you could be using this during a reorg when an asset has been deleted.

Reply:
+1